src/main/java/core/basesyntax/shape/Square.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.shape;
+
+import core.basesyntax.type.Color;
+
+public class Square extends Shape {
+    private double side;
```

Comment:
Consider making the 'side' field final, because it's never changed after the object is created.